json-schema.gemspec

```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "webmock"
   s.add_development_dependency "bundler"
 
-  s.add_runtime_dependency "addressable", '~> 2.3'
+  s.add_runtime_dependency "addressable", '~> 2.3', '>= 2.3.7'
```

Comment:
Why not directly ``'~> 2.3.7'``, is that not possible or does it have a different meaning?